### PR TITLE
Eloqua connector for Bulk API

### PR DESCRIPTION
**Pull request by kesavany**
Changed the integration tests.
Add a parameter.
Add messageType headers.

### CODE REVIEW

File: eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/accounts/createAccountImport.xml [hunk 1]
```diff
@@ -74,6 +76,7 @@
 			var updatedBy = mc.getProperty('uri.var.updatedBy');
             var updateRule = mc.getProperty('uri.var.updateRule');
```
Comment (keerthu) [inline eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/accounts/createAccountImport.xml]: Reformat the code

Comment (kesavany) [inline eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/accounts/createAccountImport.xml]: fixed in https://github.com/wso2/esb-connectors/pull/406

File: eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/accounts/createAccountImport.xml [hunk 2]
```diff
@@ -111,6 +115,9 @@
 			if (uri != null && uri != "") {
                payload["uri"]=uri;
             }
+            if (isSyncTriggeredOnImport != null && isSyncTriggeredOnImport != "") {
+               payload["isSyncTriggeredOnImport"]=isSyncTriggeredOnImport;
```
Comment (Shakila) [inline eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/accounts/createAccountImport.xml]: When doing code indentation, you must use spaces (4).

File: eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/customObjects/createCustomObjectImport.xml [hunk 3]
```diff
@@ -113,6 +117,9 @@
 			if (uri != null && uri != "") {
                payload["uri"]=uri;
             }
+            if (isSyncTriggeredOnImport != null && isSyncTriggeredOnImport != "") {
+               payload["isSyncTriggeredOnImport"]=isSyncTriggeredOnImport;
```
Comment (Shakila) [inline eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/customObjects/createCustomObjectImport.xml]: Seems not formatted correctly. Leave a space before and after the equal sign (=).

Comment (kesavany) [inline eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/customObjects/createCustomObjectImport.xml]: Fixed in https://github.com/wso2/esb-connectors/pull/406

File: eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/accounts/createAccountImport.xml (hunk 2 above)
Comment (kesavany) [inline eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/accounts/createAccountImport.xml]: Fixed in https://github.com/wso2/esb-connectors/pull/406

**Pull request merged**